src/components/DetailViewHeader.jsx

```diff
@@ -29,26 +29,28 @@ const styles = (theme) => ({
 
 export class DetailViewHeader extends React.PureComponent {
   static propTypes = {
-    addCustodian: PropTypes.func,
+    addSteward: PropTypes.func,
     addReader: PropTypes.func,
```

Comment:
if you wanna fix these linting warnings since you're touching this file, i would appreciate 🙂

Reply:
ah touché.  Will do